go/interpreter/lfvm/memory_test.go

```diff
@@ -47,3 +50,61 @@ func TestExpansionCosts(t *testing.T) {
 		}
 	}
 }
+
+func TestMemory_SetWord(t *testing.T) {
+
+	tests := map[string]struct {
+		offset uint64
+		memory []byte
+	}{
+		// test for memory expansion
```

Comment:
these comments dont tell much, the name of the test should say what it does.

Reply:
I think it will be better to use the comments as test names, as you mentioned